### PR TITLE
Baseline RFD XSD + WSDL files

### PR DESCRIPTION
**Pull request by mkotelba**
- Added RFD XSD file (originally retrieved from revision c69022faaaca06bc1af0e58a98ebc66861548dee of the erl-ihe-rfd project; see https://code.imphub.org/projects/IHE/repos/erl-ihe-rfd/).
- Added RFD WSDL files from the working copy tip of the SDCCT project.

### CODE REVIEW

File: wsdl/rfd/RFDFormArchiver.wsdl [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <wsdl:documentation>Corresponds to Transaction ITI-36 of the IHE Technical Framework.</wsdl:documentation>
+            <wsdl:input name="ArchiveForm_Message" message="ihe:ArchiveForm_Message"
+                wsaw:Action="urn:ihe:iti:2007:ArchiveForm"/>
+            <wsdl:output name="ArchiveFormResponse_Message" message="ihe:ArchiveFormResponse_Message"
+                wsaw:Action="urn:ihe:iti:2007:ArchiveFormResponse"/>
+        </wsdl:operation>
+    </wsdl:portType>
+    
+    <wsdl:binding name="RFDFormArchiver_Binding_Soap12" type="ihe:RFDFormArchiver_PortType">
+        <soap12:binding style="document" transport="http://www.w3.org/2003/05/soap/bindings/HTTP/"/>
```
Comment (mkotelba) [inline wsdl/rfd/RFDFormArchiver.wsdl:41]: SOAP HTTP transport URI corrected to that for SOAP v1.2.

File: wsdl/rfd/RFDFormArchiver.wsdl [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <wsdl:operation name="RFDFormArchiver_ArchiveForm">
+            <wsdl:documentation>Corresponds to Transaction ITI-36 of the IHE Technical Framework.</wsdl:documentation>
+            <wsdl:input name="ArchiveForm_Message" message="ihe:ArchiveForm_Message"
+                wsaw:Action="urn:ihe:iti:2007:ArchiveForm"/>
+            <wsdl:output name="ArchiveFormResponse_Message" message="ihe:ArchiveFormResponse_Message"
+                wsaw:Action="urn:ihe:iti:2007:ArchiveFormResponse"/>
+        </wsdl:operation>
+    </wsdl:portType>
+    
+    <wsdl:binding name="RFDFormArchiver_Binding_Soap12" type="ihe:RFDFormArchiver_PortType">
```
Comment (mkotelba) [inline wsdl/rfd/RFDFormArchiver.wsdl:40]: All SOAP v1.1 bindings removed - RFD specifies SOAP v1.2 only.

File: wsdl/rfd/RFDFormArchiver.wsdl [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wsdl:definitions
+    name="RFDFormArchiver"
+    targetNamespace="urn:ihe:iti:rfd:2007"
+    xmlns="http://schemas.xmlsoap.org/wsdl/"
```
Comment (mkotelba) [inline wsdl/rfd/RFDFormArchiver.wsdl:5]: RFD precisely specifies exact XML namespace prefixes (not just URIs), despite the fact that they are effectively irrelevant; whatever, complying :)

File: wsdl/rfd/RFDFormArchiver.wsdl [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    xmlns:soap12="http://schemas.xmlsoap.org/wsdl/soap12/"
+    xmlns:wsaw="http://www.w3.org/2006/05/addressing/wsdl"
+    xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/"
+    xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    
+    <wsdl:documentation>IHE Retrieve Form for Data Capture (RFD) WSDL definitions for FormArchiver.</wsdl:documentation>
+    
+    <wsdl:types>
+        <xsd:schema>
+            <xsd:import namespace="urn:ihe:iti:rfd:2007" schemaLocation="../../schema/rfd/RFD.xsd"/>
```
Comment (mkotelba) [inline wsdl/rfd/RFDFormArchiver.wsdl:16]: - RFD precisely specifies the RFD schema location; again, irrelevant.
- RFD specification ignored -> schema location modified to match the relative path in the repository (purely for the sake of convenience).

File: schema/rfd/RFD.xsd [hunk 5]
```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mkotelba) [inline schema/rfd/RFD.xsd:1]: Unmodified from the original version.

File: wsdl/rfd/RFDFormArchiver.wsdl [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    <wsdl:message name="ArchiveFormResponse_Message">
+        <wsdl:documentation>Archive Form response message.</wsdl:documentation>
+        <wsdl:part name="body" element="ihe:ArchiveFormResponse"/>
+    </wsdl:message>
+    
+    <wsdl:portType name="RFDFormArchiver_PortType">
+        <wsdl:operation name="RFDFormArchiver_ArchiveForm">
+            <wsdl:documentation>Corresponds to Transaction ITI-36 of the IHE Technical Framework.</wsdl:documentation>
+            <wsdl:input name="ArchiveForm_Message" message="ihe:ArchiveForm_Message"
+                wsaw:Action="urn:ihe:iti:2007:ArchiveForm"/>
```
Comment (mkotelba) [inline wsdl/rfd/RFDFormArchiver.wsdl:34]: - WS Addressing WSDL schema namespace URI fixed.
- WSA Action URIs fixed/added to comply with the RFD specification.

**Comment by matthew-chudy** [comment]
Without going through the process of updating my FormManager application to test,  I believe that these changes will meed our needs.

**Pull request merged**